cmd/ack-generate/command/controller.go

```diff
@@ -226,3 +236,43 @@ func getLatestAPIVersion() (string, error) {
 	})
 	return versions[len(versions)-1], nil
 }
+
+// FallBackFindServiceID reads through aws-sdk-go/models/apis/*/*/api-2.json
+// Returns ServiceID (as newSuppliedAlias) if supplied service Alias matches with serviceID in api-2.json
+// If not a match, return the supllied alias.
+func FallBackFindServiceID(sdkDir, svcAlias string) (string, error) {
```

Comment:
we could potentially hide this as a non-exported method on the `pkg/model.SDKHelper` struct that gets triggered here: https://github.com/aws/aws-controllers-k8s/blob/57ddece0bb878956e9be5d111e562b7959522b9a/pkg/model/sdk_helper.go#L63-L65

But, I can work on that in a followup PR. I just tested this locally and `make test` after running `make build-controller SERVICE=sfn` works like a charm. :)

Reply:
Ah! yes. Should I do this @jaypipes ?

Reply:
@ChaturvediSulabh yes, if you'd like to do that, please go ahead!

Reply:
Sure, I'll do this.